lighthouse-core/scripts/build-report-for-autodeployment.js

```diff
@@ -29,19 +39,22 @@ const DIST = path.join(LH_ROOT, `dist/now`);
     'ɑrabic': swapLocale(lhr, 'ar').lhr,
     'xl-accented': swapLocale(lhr, 'en-XL').lhr,
     'error': errorLhr,
+    'soloperf': tweakLhrForPsi(lhr),
```

Comment:
this name doesn't mean anything to me. what is it? would `psi` work?

Reply:
it should actually be `solocat`. does that have more meaning? i really just mean to create a report where there's only 1 category (as we  have special logic to handle it)

Reply:
`single-category`. readable :)